examples/tts/data_baker/local/prepare_data.py

```diff
@@ -270,10 +280,13 @@ def convert_audio_and_split_transcript(dataset_dir, total_csv_path):
         for line in f:
             transcript_id = line.strip().split("|")[0]
             transcript = line.strip().split("|")[1]
+            #downsample 48k to 24k
+            wav48k = os.path.join(audio_dir, 'Wave/' + transcript_id + '.wav')
+            wav24k = os.path.join(audio_dir, 'Wave_24/' + transcript_id + '.wav')
```

Comment:
do we really need to do the downsample?

Reply:
the performance using 48000 sample rate audio remains unknown. maybe need more experiments.